Bio/SeqRef/_commons.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 by Tianyi Shi.  All rights reserved.
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+
+class SeqId(object):
```

Comment:
Why put this in a private module, and not directly in the public API for ``Bio.SeqRef``?